sdk/python/kfp/_client.py

```diff
@@ -139,6 +141,14 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow', other_client
     self._experiment_api = kfp_server_api.api.experiment_service_api.ExperimentServiceApi(api_client)
     self._pipelines_api = kfp_server_api.api.pipeline_service_api.PipelineServiceApi(api_client)
     self._upload_api = kfp_server_api.api.PipelineUploadServiceApi(api_client)
+    self._healthz_api = kfp_server_api.api.healthz_service_api.HealthzServiceApi(api_client)
+    if not self._context_setting['namespace'] and self.get_kfp_healthz().multi_user is True:
+      try:
+        with open(Client.NAMESPACE_PATH, 'r') as f:
+          current_namespace = f.read()
+          self.set_user_namespace(current_namespace)
+      except:
```

Comment:
Maybe you could try to only catch the exact type of error that you expect here

Reply:
also we should log the exception in logging.info

Reply:
I agree, however, due to my limited experience with programming and no ability to test this PR fully on my own cluster easily I'm not 100% sure what exception can all be expected. The main one I had in mind is `FileNotFoundError` I believe. So I have added that and added `exc_info=True` to the logging.info line. Hopefully this is correct.

Reply:
Testing just passed so I believe this can be considered solved.